.github/CONTRIBUTING.md

```diff
@@ -418,26 +418,26 @@ for(var/atom/thing as anything in bag_of_atoms)
 
-* All changes to the database's layout(schema) must be specified in the database changelog in SQL, as well as reflected in the schema files
+* All changes to the database's layout(schema) should be specified in the database changelog in SQL, as well as reflected in the schema files
 
-* Any time the schema is changed the `schema_revision` table and `DB_MAJOR_VERSION` or `DB_MINOR_VERSION` defines must be incremented.
+* Any time the schema is changed the `schema_revision` table and `DB_MAJOR_VERSION` or `DB_MINOR_VERSION` defines should be incremented.
 
-* Queries must never specify the database, be it in code, or in text files in the repo.
+* Queries should never specify the database, be it in code, or in text files in the repo.
 
-* Primary keys are inherently immutable and you must never do anything to change the primary key of a row or entity. This includes preserving auto increment numbers of rows when copying data to a table in a conversion script. No amount of bitching about gaps in ids or out of order ids will save you from this policy.
+* Primary keys are inherently immutable and you should never do anything to change the primary key of a row or entity. This includes preserving auto increment numbers of rows when copying data to a table in a conversion script. No amount of bitching about gaps in ids or out of order ids will save you from this policy.
 
-* The ttl for data from the database is 10 seconds. You must have a compelling reason to store and reuse data for longer then this.
+* The ttl for data from the database is 10 seconds. You should have a compelling reason to store and reuse data for longer then this.
 
 * Do not write stored and transformed data to the database, instead, apply the transformation to the data in the database directly.
 	* ie: SELECTing a number from the database, doubling it, then updating the database with the doubled number. If the data in the database changed between step 1 and 3, you'll get an incorrect result. Instead, directly double it in the update query. `UPDATE table SET num = num*2` instead of `UPDATE table SET num = [num]`.
 	* if the transformation is user provided (such as allowing a user to edit a string), you should confirm the value being updated did not change in the database in the intervening time before writing the new user provided data by checking the old value with the current value in the database, and if it has changed, allow the user to decide what to do next.
 
 ### Mapping Standards
 * Adding, Removing, or Replacing Station Maps
-	* All pull requests adding, removing, or replacing station maps must receive prior approval from a maptainer, or they will be closed without additional warning.
+	* All pull requests adding, removing, or replacing station maps should receive prior approval from a maptainer, or they will be closed without additional warning.
```

Comment:
Leaving the door open to disappointment, they should really have already been in discussions with a maptainer much earlier in the process than even this